packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -241,7 +241,7 @@ const BlockComponent = forwardRef(
 				// Overrideable props.
 				aria-label={ blockLabel }
 				role="group"
-				{ ...omit( wrapperProps, [ 'data-align' ] ) }
```

Comment:
This is like this because the attribute is now applied to a wrapper. This change might be causing the breakage @Addison-Stavlo noticed.

Reply:
You mean the `<div className="wp-block" data-align="..." />` wrapper? But as of #23089, that's added by `block-list/block.js` rather than `block-wrapper.js`. And AFAICS, `block-wrapper` omitting that prop when instantiating the block component is why that wrapper wasn't added. Or am I missing something?

Reply:
https://github.com/WordPress/gutenberg/blob/332397e9552bed0454704f005b857e5ba8de4b9f/packages/block-editor/src/components/block-list/block.js#L160-L169

I see it on a different `div` there.

Reply:
But [that's](https://github.com/WordPress/gutenberg/blob/332397e9552bed0454704f005b857e5ba8de4b9f/packages/block-editor/src/components/block-list/block.js#L160-L169) the one I'm talking about:

> You mean the `<div className="wp-block" data-align="..." />` wrapper?

Try this, on `master`, and then on this branch:

```diff
diff --git a/packages/block-editor/src/components/block-list/block.js b/packages/block-editor/src/components/block-list/block.js
index 93ae691015..aca5b8bb8f 100644
--- a/packages/block-editor/src/components/block-list/block.js
+++ b/packages/block-editor/src/components/block-list/block.js
@@ -95,6 +95,10 @@ function BlockListBlock( {
        );
        const isUnregisteredBlock = name === getUnregisteredTypeHandlerName();
 
+       if ( name === 'core/cover' ) {
+               console.log( { wrapperProps } );
+       }
+
        // Determine whether the block has props to apply to the wrapper.
        if ( blockType.getEditWrapperProps ) {
                wrapperProps = {
```

`master`:

![wrapper-props-master](https://user-images.githubusercontent.com/96308/86008231-c4165200-ba18-11ea-98b8-49b5f5dbd5e8.png)

This branch:

![wrapper-props-branch](https://user-images.githubusercontent.com/96308/86008214-be207100-ba18-11ea-8084-8fa9040da1b0.png)

Reply:
The block wrapper is a child in this file. How would omitting the prop there make it not show here?

https://github.com/WordPress/gutenberg/blob/332397e9552bed0454704f005b857e5ba8de4b9f/packages/block-editor/src/components/block-list/block.js#L207-L212

`blockEdit` is already wrapped at that point.

Reply:
Right. I was getting my wires crossed here with regard to which component uses which (and I could've sworn I re-built Gutenberg with and without the change to verify, but apparently not :man_shrugging: )

Anyway, I've reverted the relevant commit, and Twenty Twenty Blocks still looks as before reverting :+1: However, Twenty Nineteen Blocks also still looks garbled :confused:

Reply:
Sounds like a different issue then?

Reply:
Yeah, the relevant styling is coming from https://github.com/WordPress/theme-experiments/blob/2abc1c0b025ad966e1ee9fd6ff5f4029e9f3900b/twentynineteen-blocks/twentynineteen-styles/style-editor.css#L578-L583